src/main/java/cms/gov/madie/measure/resources/ValidationController.java

```diff
@@ -28,14 +32,25 @@ public class ValidationController {
 
   private FhirServicesClient fhirServicesClient;
   private VirusScanClient virusScanClient;
+  private ObjectMapper mapper;
 
   @PostMapping(
       path = "/bundles",
       consumes = MediaType.APPLICATION_JSON_VALUE,
       produces = MediaType.APPLICATION_JSON_VALUE)
   public ResponseEntity<String> validateBundle(
       HttpEntity<String> request, @RequestHeader("Authorization") String accessToken) {
-    return fhirServicesClient.validateBundle(request.getBody(), accessToken);
+    try {
+      ResponseEntity<HapiOperationOutcome> output =
+          fhirServicesClient.validateBundle(request.getBody(), accessToken);
+      return ResponseEntity.ok(mapper.writeValueAsString(output.getBody()));
+
+    } catch (JsonProcessingException ex) {
+      return ResponseEntity.badRequest()
+          .body(
```

Comment:
May be a test case to cover this scenario and Looks like Codacy is failing some checks